book/src/configuration.md

```diff
@@ -11,8 +11,37 @@ This example file demonstrates the default configuration:
 [general]
 # Directories to search for templates, relative to the crate root.
 dirs = ["templates"]
+# Unless you add a `-` in a block, whitespace won't be trimmed.
+automatic_trim_block = false
 ```
 
+About `automatic_trim_block`, if you want to trim whitespace before or after a
```

Comment:
This sentence is a bit awkward. Maybe something like, "In the default configuration, you can use the `-` operator to indicate that whitespace should be suppressed before or after a block. For example:".